docs/manufacturer/cloud-connector/index.md

```diff
@@ -56,8 +54,8 @@ Unzip the archive on your machine and follow the step. You can also find those s
 Before starting the installation make sure to prepare a CloudConnector configuration for the target machine to have all OPC UA server settings in place.
 
 ```console
-user@example:~$ tar xzvf Tapio.CloudConnector.<platform>.tar.gz
-user@example:~$ cd Tapio.CloudConnector.Service.Core
+user@example:~$ tar xzvf tapio.cloudConnector.<platform>.tar.gz
```

Comment:
müsste afaik 'tapio.cloudconnector' sein. Aber check das bitte nochmal mit dem file gegen

Reply:
ah thanks, fixed it.